bot/cogs/commands.py

```diff
@@ -27,8 +31,9 @@ def __init__(self, bot):
         self.logger = logging.Logger("COMMANDS")
 
     @commands.Cog.listener()
-    async def on_command_error(self, ctx, error):
+    async def on_command_error(self, ctx: commands.Context, error):
         # We get an error about commands being found when using "runtime" commands, so we have to ignore that
+        self.logger.error(f"Caught {error!r}, {dir(error)}")
```

Comment:
This will error with every message Fred reads

Reply:
It's a logger error only, so that we can find names of uncaught exceptions.